backend/ordd_api/migrations/0017_migrate_content_to_v10.py

```diff
@@ -0,0 +1,266 @@
+    item.name = 'Base data'
+    item.save()
+
+    #    KeyTag
+    item = KeyTag.objects.using(db_alias).get(name='Road infrastructure')
+    item.name = 'Roads'
+    item.save()
+
+    item = KeyTag.objects.using(db_alias).get(name='Harbours')
+    item.name = 'Harbors'
+    item.save()
+
+    #    KeyDatasetName
+    item = KeyDatasetName.objects.using(db_alias).get(name='Hazard Scenarios')
+    item.name = 'Historical records of hazard events'
+    item.save()
+
+    #    KeyDatasetName
+    item = KeyDatasetName.objects.using(db_alias).get(
+        name='List Type Of Volcanoes')
+    item.name = 'Volcanoes'
+    item.save()
+
+    #    KeyDatasetName simple capitalization
+    items = KeyDatasetName.objects.using(db_alias).all()
+    for item in items:
+        item.name = item.name.capitalize()
+        item.save()
+
+    kd = []
+    kd_code = []
+    kt = []
+
+    with open('contents/key_datasets/resilience-index-'
+              'datasets-list-v10 - Datasets.csv', 'r') as kdfile,\
+        open('contents/key_datasets/resilience-index-'
+             'datasets-list-v10 - Tags.csv', 'r') as ktfile:
+        kdcsv = csv.reader(kdfile, delimiter=',')
+        ktcsv = csv.reader(ktfile, delimiter=',')
+
+        for row in kdcsv:
+            if len(row) == 0:
+                continue
+            kd.append(row)
+            kd_code.append(row[0])
+        kd = kd[1:]
+        kd_code = kd_code[1:]
+
+        for row in ktcsv:
+            if len(row) == 0:
+                continue
+            kt.append(row)
+        kt = kt[1:]
+
+        #
+        #
+        #     CHECKING AND INSERTING
+        #
+        #
+
+        #
+        #  INSERT MISSING DATASETNAME
+        #
+        print("  INSERT MISSING DATASETNAME")
+        for keydataset in kd:
+            datasetname = keydataset[2]
+            print(datasetname)
+            try:
+                item = KeyDatasetName.objects.using(db_alias).get(
+                    name=datasetname)
+            except KeyDatasetName.DoesNotExist:
+                print("DatasetName [%s] is missing, create it" % datasetname)
+                KeyDatasetName.objects.using(
+                    db_alias).create(name=datasetname)
+
+        #
+        #  CHECK MISSING CATEGORY
+        #
+        print("  CHECK MISSING CATEGORY")
+        for keydataset in kd:
+            category = keydataset[1]
+            print("  check [%s]" % category)
+            item = KeyCategory.objects.using(db_alias).get(
+                name=category)
+
+        #
+        #  CHECK MISSING TAGGROUPS
+        #
+        print("  CHECK MISSING TAGGROUPS")
+        for keydataset in kd:
+            taggroup = keydataset[3]
+            print("  check [%s]" % taggroup)
+            if taggroup == "":
+                continue
+            # try:
+            item = KeyTagGroup.objects.using(db_alias).get(
+                name=taggroup)
+            # except KeyTagGroup.DoesNotExist:
```

Comment:
This kind of exception does never happens?

Reply:
If it happens is correct that it breaks, remove the commented part at all.